### PR TITLE
Create XGBoost.py

### PR DESCRIPTION
**Pull request by joshi1411**
Extreme Gradient Boosting Algorithm implementation using sklearn.

### CODE REVIEW

**Comment by cclauss** [comment]
Automated tests are failing.
